classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.Classroom = self
+  end
+end
```

Comment:
- [ ] Let's have `attr_reader :students` for this file as you're reading students when adding a student to a classroom. 👍🏽

Reply:
Hey @AmaduKamara, thanks for the feedback. I have now implemented the change